runtests.py

```diff
@@ -357,7 +357,10 @@ def build_project(args):
                 '-Werror=unused-function',
             ])
             env['CFLAGS'] = warnings_as_errors + ' ' + env.get('CFLAGS', '')
+
+    cmd += ["build"]
     if args.debug or args.gcov:
+        cmd += ["--debug"]
         # assume everyone uses gcc/gfortran
         env['OPT'] = '-O0 -ggdb'
```

Comment:
what the point of using `-ggdb` over `-g`?

Reply:
Probably gdb format was more useful than OS native format in the past. However it's 2020 now, and assuming everyone uses GCC seems a little outdated.

Reply:
make sense thanks!, I kept it untouched, I will try to release a cleanup patch to beautify build logs and remove such cases like this one.